satpy/readers/viirs_vgac_l1c_nc.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2009-2019 Satpy developers
```

Comment:
That was four years ago.

Reply:
Thanks for spotting the copy-paster errors! I will fix them :)!